README.md

```diff
@@ -2,8 +2,11 @@
 <p align="center">
     <p align="center"> <img title="BPMN Visualization" src="docs/images/diagram-hacktoberfest-example.png"></p>
     <p align="center"> 
+        <a href="https://npmjs.org/package/bpmn-visualization">
+          <img alt="npm package" src="https://img.shields.io/npm/v/bpmn-visualization.svg?color=blue"> 
```

Comment:
```suggestion
          <img alt="npm package" src="https://img.shields.io/npm/v/bpmn-visualization.svg?color=orange"> 
```

As normally, the release version & the npm version are the same, I'll use the same color.